hoomd/hpmc/pytest/test_external_wall.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Test hoomd.hpmc.external.wall."""
```

Comment:
Add tests that place particles and check that they overlap (or not overlap) with walls.

Reply:
Done